source/slang/slang-reflection.cpp

```diff
@@ -86,6 +86,15 @@ static inline SlangReflectionEntryPoint* convert(EntryPointLayout* entryPoint)
     return (SlangReflectionEntryPoint*) entryPoint;
 }
 
+static inline EntryPointGroupLayout* convert(SlangEntryPointGroupLayout* entryPointGroup)
```

Comment:
Not an asInternal?

Reply:
The old functions that support the C-style API have been left as-is with their original naming.